packages/tools/src/tools/annotation/BidirectionalTool.ts

```diff
@@ -1172,8 +1172,8 @@ export default class BidirectionalTool extends AnnotationTool {
     // spaceBetweenSlices & pixelSpacing &
     // magnitude in each direction? Otherwise, this is "px"?
     const textLines = [
-      `L: ${length.toFixed(2)} ${unit}`,
-      `W: ${width.toFixed(2)} ${unit}`,
+      `L: ${length.toFixed(2)} ${unit ?? 'mm'}`,
```

Comment:
This should probably be 'px', as the default assumption is that units are in pixel values, not that they are in mm.  That is a patient safety issue, because an assumption of mm when it is actually px means the user could assume it has been calibrated, and use it, eg for surgery.  At a minimum, I would import the default unit from a common place.